src/feature/common/theme.ts

```diff
@@ -26,10 +26,11 @@ export const darkTheme: MantineThemeOverride = {
     modal: ["#191919", "#111111"],
     title: ["#ffffff"],
     text: ["#bcc2c4"],
-    accent1: ["#99f5ff", "#48abe0"],
-    accent2: ["#7b61ff", "#7418b5"],
+    accent1: ["#99f5ff", "#48abe0", "#d4fbff"],
```

Comment:
![image](https://github.com/flojoy-io/studio/assets/47371088/f253bc7a-4e56-49d7-a9ce-b7fa5653d6c9)
This button probably shouldn't be white on hover.

Reply:
@JeffDotPng It was the color on flojoy.io/brand for the button hover. Would you like me to change it to a lighter color? Or change the color of the settings button instead?

Reply:
Imo just change the hover color to a lighter cyan color like the accent (just lower the opacity on one of the accent colors), this current one is very blinding for dark mode.

Reply:
@JeffDotPng Would this be better color??

<img width="59" alt="Screenshot 2023-05-30 at 8 58 49 PM" src="https://github.com/flojoy-io/studio/assets/89884098/ed73bc0c-656e-4981-b9dd-809489b1d951">

Reply:
@JeffDotPng  This is what the setting button looks like right now. Let me know if there is anything else to change!

<img width="75" alt="Screenshot 2023-05-30 at 9 38 31 PM" src="https://github.com/flojoy-io/studio/assets/89884098/1bf3ebca-fa4c-431e-97f7-12fdd6314ec8">
<img width="71" alt="Screenshot 2023-05-30 at 9 38 42 PM" src="https://github.com/flojoy-io/studio/assets/89884098/bb674318-e566-4a8d-81b0-dacf2cc63a8f">